cmd/mattermost/commands/command.go

```diff
@@ -67,3 +89,80 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func createCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
```

Comment:
You can use the `cobra.MinimunNArgs` restriction to do exactly that. An example here: https://github.com/mattermost/mattermost-server/pull/9512/files#diff-6889f01f236fa4b8a3050c7954ec0b68R68

Reply:
will do